public/assets/sass/pages/get-involved/clubs.scss

```diff
@@ -0,0 +1,74 @@
+@import "../../util/_sass-mixins";
+@import "../../util/variables";
+@import "../../mixins/banner-image";
+
+section {
+  max-width: var(--content-width);
+  margin: 30px auto;
+  padding-left: 0;
+  position: relative;
+
+  .stepper {
+    margin-top: 50px;
+    --size: 3.5rem;
+
+    .stepper_item {
+      margin: 0;
+      position: relative;
+      display: flex;
+      gap: 1rem;
+      padding-bottom: 2rem;
+      pointer-events: none;
+    }
+    .stepper_item::before {
+      content: "";

```

Comment:
is this line necessary? not sure about your logic here but if its necessary, feel free to resolve

Reply:
yes - its some CSS requirement such that the background isnt visible if there's no content, even if its an empty string. the discord access SCSS uses the same rule